MetaCocktailsSwiftData/Cocktails/Classics/Caipirinha.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Caipirinha.swift
+//  MetaCocktailsSwiftData
+//
+//  Created by James on 10/8/23.
+//
+
+import SwiftUI
+
+var caipirinha   = Cocktail(cocktailName: "Caipirinha",
+                            glasswareType: .doubleOld,
+                            garnish: [.noGarnish],
+                            ice: .koldDraft,
+                            author: "Brazil",
+                            spec: caipirinhaSpec,
+                            buildOrder: caipirnhaBuild,
+                            tags: caipirnhaTags)
+
+
+var caipirinhaSpec = [CocktailIngredient(.lime, value: 2, unit: .limeQuarters),
+                      CocktailIngredient(.granulatedSugar , value: 1.5, unit: .teaspoons),
+                      CocktailIngredient(.cachaca, value: 2, unit: .fluidOunces)]
+                      
```

Comment:
Not a big deal, but remeber that .fluidOunces is the default so you this can just be:

`CocktailIngredient(.cachaca, value: 2)`